lib/types/geoman-minimal.d.ts

```
@@ -0,0 +1,120 @@
+
+  // Map events
+  interface MapEvents {
+    "pm:create": (e: {
+      layer: Layer;
+      layerType?: string;
+      shape?: string;
+    }) => void;
+    "pm:remove": (e: { layer: Layer }) => void;
+    "pm:update": (e: { layer: Layer }) => void;
+    "pm:cut": (e: { layer?: Layer; layers?: Layer[] }) => void;
+    "pm:undo": (e: any) => void;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)